spanner/tests/system/test_system.py

```diff
@@ -488,6 +488,35 @@ def test_transaction_read_and_insert_then_rollback(self):
         self.assertEqual(rows, [])
 
     @RetryErrors(exception=GrpcRendezvous)
+    def test_transaction_read_and_insert_then_execption(self):
+        retry = RetryInstanceState(_has_all_ddl)
+        retry(self._db.reload)()
+
+        class TestException(Exception):
+            pass
```

Comment:
3451e47

>why not just use a RuntimeError?

The Java systest specifically names 'user-defined exception" as the trigger.